src/app.ts

```diff
@@ -41,7 +41,7 @@ export default class App {
     // Error handling
     this.app.use(serverErrorHandler);
 
-    const port = ConfigService.getIntValue('server.port');
+    const port = process.env.PORT || ConfigService.getIntValue('server.port');
```

Comment:
Why is this needed? We added `server.port` in custom config.

Reply:
yes @akashlakhera  sir, actually here we are trying to get the integer value of port defined in the environment, But on heroku, the value of port is in the form of string therefore we were always getting value mismatch type error, 
so In order to remove that error I have added this line, if the port is defined in environment then it will take value using this ```process.env.PORT```

Reply:
Then we can do a `getStringValue`. I think every platform/infra provider (AWS, DO, etc) will provide env vars in string.

Reply:
ok @akashlakhera  sir, I think that would work.